domain-server/resources/web/settings/js/settings.js

```diff
@@ -711,8 +713,8 @@ $(document).ready(function(){
       name: 'places',
       label: 'Places',
       html_id: Settings.PLACES_TABLE_ID,
-      help: "The following places currently point to this domain.</br>To point places to this domain, "
-        + " go to the <a href='" + METAVERSE_URL + "/user/places'>My Places</a> "
+      help: "To point places to this domain, "
+        + " go to the <a href='" + METAVERSE_URL + "/user/places'>Places</a> "
```

Comment:
Should this URL be https://dashboard.vircadia.com/places?

Reply:
The metaverse API will redirect you when you hit that link, so the metaverse API can specify where its dashboard is.